### PR TITLE
revert-quickstarts links

### PR DESCRIPTION
**Pull request by ObedVega**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ function HomepageHeader() {
         <Banner
             content={
               <div>
-                <Typography scale="body1">A scalable, on-demand compute engine in the cloud.</Typography> 
+                <Typography scale="body1">Your go-to hub for quickstarts, guides, tutorials, code samples, technical documentation, and downloads for all Teradata products.</Typography>
```
Comment (owilliams320) [inline src/pages/index.js:22]: Can you make sure this is pulling from translations

File: src/pages/index.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ function HomepageHeader() {
               </div>}
             imageSrc={HeroImageUrl}
             isHero={true}
-            title="Welcome to the Teradata Developers Portal"
+            title="Welcome to the Teradata Developer Portal"
```
Comment (owilliams320) [inline src/pages/index.js:35]: Can you make sure this is pulling from translations

**Review by owilliams320 — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 1 above)
Comment (ObedVega) [reply src/pages/index.js:22]: Done
<Banner
            content={
              <div>
                <Typography scale="body1">{translate({ message: 'developers.title_description' })}</Typography> 
                <br/>
                <Link to="https://quickstarts.teradata.com/">
                  <Button
                    label="Learn more"
                    primary={true}
                    icon="fa fa-arrow-right"
                    trailingIcon={true}
                  />
                </Link>
              </div>}
            imageSrc={HeroImageUrl}
            isHero={true}
            title={translate({ message: 'developers.welcome_message' })}
          />

File: src/pages/index.js (hunk 2 above)
Comment (ObedVega) [reply src/pages/index.js:35]: Done
<Banner
            content={
              <div>
                <Typography scale="body1">{translate({ message: 'developers.title_description' })}</Typography> 
                <br/>
                <Link to="https://quickstarts.teradata.com/">
                  <Button
                    label="Learn more"
                    primary={true}
                    icon="fa fa-arrow-right"
                    trailingIcon={true}
                  />
                </Link>
              </div>}
            imageSrc={HeroImageUrl}
            isHero={true}
            title={translate({ message: 'developers.welcome_message' })}
          />

**Review by owilliams320 — APPROVED** [review]

**Pull request merged**